client/app/expenses/components/ExpenseItem.tsx

```diff
@@ -0,0 +1,52 @@
+import { ItemDisplay } from './ItemDisplay'
+import { PaymentItem } from './PaymentItem'
+import { UserDisplay } from './UserDisplay'
+import { ExpenseDate } from '../types'
+
+type ExpenseItemProps = {
+  expense: ExpenseDate
+  userId: number
+}
+
+const ExpenseItem = ({ expense, userId }: ExpenseItemProps) => {
```

Comment:
I think this is better design because it's kinda uneven the line now!
* Edit/Delete buttons below the item
* The item name and price should be more obvious (600, 16px for SP)
* If you change like the above, you need a little bit padding-top for item/price 

Also, you can make the fors for Date and user name bigger for SP!

<img width="904" alt="Screenshot 2024-04-09 at 3 05 25 PM" src="https://github.com/HAK2024/fairy-share/assets/66394413/1b4af560-adf0-4bc4-967b-ad1c18f719cd">

<img width="352" alt="Screenshot 2024-04-09 at 3 04 53 PM" src="https://github.com/HAK2024/fairy-share/assets/66394413/c5622ca8-6bfe-498f-9923-3cc9868eeeee">

Reply:
Fixed!
https://github.com/HAK2024/fairy-share/pull/113/commits/800ef0f1eb10ff8ccb9dd9b3b8cfaaa2e6593931